### PR TITLE
plugins:foreman_cockpit - Release 1.0.2

### PR DESCRIPTION
**Pull request by dLobatog**
Same as the current spec in develop, except for the scl_prefix_ruby & 'tfm' prefix in comps

### CODE REVIEW

**Comment by domcleal** [comment]
Also run ./comps_doc.sh to add the -doc subpackages to comps/.

**Comment by dLobatog** [comment]
@domcleal Done, thanks!

File: comps/comps-foreman-plugins-rhel7.xml [hunk 1]
```diff
@@ -165,10 +167,8 @@
       <packagereq type="default">rubygem-smart_proxy_abrt-doc</packagereq>
       <packagereq type="default">rubygem-smart_proxy_chef-doc</packagereq>
       <packagereq type="default">rubygem-smart_proxy_discovery-doc</packagereq>
-      <packagereq type="default">rubygem-smart_proxy_dynflow-doc</packagereq>
```
Comment (dLobatog) [inline comps/comps-foreman-plugins-rhel7.xml]: Not sure why these are removed?

Comment (domcleal) [inline comps/comps-foreman-plugins-rhel7.xml]: Oh, probably because they've been cherry picked back.  Sorry, please leave them in!

**Comment by domcleal** [comment]
Thanks @dLobatog, building and I'll try to retag into 1.8.

**Pull request merged**